src/reports/common.rkt

```diff
@@ -317,23 +321,25 @@
   (-> test? string?)
   (define output-repr (test-output-repr test))
   (string-join
-   (filter
-    identity
-    (list
-     (if (test-identifier test)
-         (format "(FPCore ~a ~a" (test-identifier test) (test-vars test))
-         (format "(FPCore ~a" (test-vars test)))
-     (format "  :name ~s" (test-name test))
-     (format "  :precision ~s" (representation-name (test-output-repr test)))
-     (if (equal? (test-pre test) '(TRUE)) #f (format "  :pre ~a" (prog->fpcore (test-pre test))))
-     (if (equal? (test-expected test) #t) #f (format "  :herbie-expected ~a" (test-expected test)))
-     (and (test-output test)
-          (not (null? (test-output test)))
-          (format "\n~a"
-                  (string-join (map (lambda (exp) (format "  :alt\n  ~a\n" (car exp)))
-                                    (test-output test))
-                               "\n")))
-     (format "  ~a)" (prog->fpcore (test-input test)))))
+   (filter identity
```

Comment:
Stupid `raco fmt`. I assume no changes other than an extra argument to `prog→fpcore`?